Workflow run 'GreptimeDB Nightly Build' failed

@GreptimeTeam/db-approver
New failure: https://github.com/GreptimeTeam/greptimedb/actions/runs/14743820927

```
  cargo:warning=In file included from /tmp/grammarRxP2wc/parser.c:3:0:
  cargo:warning=/tmp/grammarRxP2wc/parser.c: In function 'ts_lex':
  cargo:warning=/tmp/grammarRxP2wc/tree_sitter/parser.h:205:5: error: 'for' loop initial declarations are only allowed in C99 mode
  cargo:warning=     for (uint32_t i = 0; i < sizeof(map) / sizeof(map[0]); i += 2) {  \
  cargo:warning=     ^
  cargo:warning=/tmp/grammarRxP2wc/parser.c:1521:7: note: in expansion of macro 'ADVANCE_MAP'
  cargo:warning=       ADVANCE_MAP(
  cargo:warning=       ^
  cargo:warning=/tmp/grammarRxP2wc/tree_sitter/parser.h:205:5: note: use option -std=c99 or -std=gnu99 to compile your code
  cargo:warning=     for (uint32_t i = 0; i < sizeof(map) / sizeof(map[0]); i += 2) {  \
  cargo:warning=     ^
  cargo:warning=/tmp/grammarRxP2wc/parser.c:1521:7: note: in expansion of macro 'ADVANCE_MAP'
  cargo:warning=       ADVANCE_MAP(
  cargo:warning=       ^
  cargo:warning=/tmp/grammarRxP2wc/tree_sitter/parser.h:205:5: error: 'for' loop initial declarations are only allowed in C99 mode
  cargo:warning=     for (uint32_t i = 0; i < sizeof(map) / sizeof(map[0]); i += 2) {  \
  cargo:warning=     ^
  cargo:warning=/tmp/grammarRxP2wc/parser.c:1655:7: note: in expansion of macro 'ADVANCE_MAP'
  cargo:warning=       ADVANCE_MAP(
  cargo:warning=       ^
  cargo:warning=/tmp/grammarRxP2wc/parser.c: At top level:
  cargo:warning=cc1: warning: unrecognized command line option "-Wno-everything" [enabled by default]

  --- stderr


  error occurred in cc-rs: command did not execute successfully (status code exit status: 1): LC_ALL="C" "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "/tmp/grammarRxP2wc" "-I" "/tmp/grammarRxP2wc/sysroot" "-Wall" "-Wextra" "-Wno-unused-label" "-Wno-unused-parameter" "-Wno-unused-but-set-variable" "-Wno-trigraphs" "-Wno-everything" "-o" "/greptimedb/target/nightly/build/dfir_datalog_core-31f288dd51f165a4/out/873dbcc71ea67789-parser.o" "-c" "/tmp/grammarRxP2wc/parser.c"


warning: build failed, waiting for other jobs to finish...
 WARN rustc_codegen_ssa::back::link The linker driver does not support `-fuse-ld=lld`. Retrying without it.
make: *** [build] Error 101
make: *** [Makefile:85: build-by-dev-builder] Error 2
```

--- a/src/flow/Cargo.toml
+++ b/src/flow/Cargo.toml
@@ -40,7 +40,7 @@ datafusion-expr.workspace = true
 datafusion-physical-expr.workspace = true
 datafusion-substrait.workspace = true
 datatypes.workspace = true
-dfir_rs = "0.13.0"
+dfir_rs = { version = "0.13.0", default-features = false }
 enum-as-inner = "0.6.0"
 enum_dispatch = "0.3"
 futures.workspace = true

